conf.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+
+extensions = []
+templates_path = ['/home/docs/checkouts/readthedocs.org/readthedocs/templates/sphinx', 'templates', '_templates', '.templates']
```

Comment:
We shouldn't have to reference directories that don't exist (on our machines). I realise this is copied from the conf.py RTD supplied. In a project generated by `sphinx-quickstart` I only got the `'_templates'` entry and the slab of config added by RTD did not alter it. (https://readthedocs.org/api/v2/build/8767050.txt)

Reply:
Ok, updated.